src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java

```diff
@@ -164,4 +170,44 @@ public static void deletePushTopic() {
     String pushTopicName = SalesforcePropertiesPageActions.topicName;
     SalesforceClient.deletePushTopic(pushTopicName);
   }
+
+  @After(order = 1, value = "@BQ_SINK_TEST")
+  public static void deleteTempTargetBQTable() throws IOException, InterruptedException {
+    String bqTargetTableName = PluginPropertyUtils.pluginProp("bqTargetTable");
+    try {
+      BigQueryClient.dropBqQuery(bqTargetTableName);
+      BeforeActions.scenario.write("BQ Target table - " + bqTargetTableName + " deleted successfully");
+      PluginPropertyUtils.removePluginProp("bqTargetTable");
+    } catch (BigQueryException e) {
+      if (e.getMessage().contains("Not found: Table")) {
+        BeforeActions.scenario.write("BQ Target Table " + bqTargetTableName + " does not exist");
+      } else {
+        Assert.fail(e.getMessage());
+      }
+    }
+  }
+
+  @After(order = 1, value = "@BQ_SINK_TEST")
```

Comment:
update the hook name as it is for multi source it can cause issues and confusion while running pipelines.

Reply:
done